server/src/test/java/org/opensearch/index/store/RemoteSegmentStoreDirectoryTests.java

```diff
@@ -697,13 +698,20 @@ public void testCleanupAsync() throws Exception {
             threadPool,
             indexShard.shardId()
         );
-        when(remoteSegmentStoreDirectoryFactory.newDirectory(any(), any(), any())).thenReturn(remoteSegmentDirectory);
+        when(remoteSegmentStoreDirectoryFactory.newDirectory(any(), any(), any(), any())).thenReturn(remoteSegmentDirectory);
         String repositoryName = "test-repository";
         String indexUUID = "test-idx-uuid";
         ShardId shardId = new ShardId(Index.UNKNOWN_INDEX_NAME, indexUUID, Integer.parseInt("0"));
-
-        RemoteSegmentStoreDirectory.remoteDirectoryCleanup(remoteSegmentStoreDirectoryFactory, repositoryName, indexUUID, shardId);
-        verify(remoteSegmentStoreDirectoryFactory).newDirectory(repositoryName, indexUUID, shardId);
+        RemoteStorePathType pathType = RemoteStorePathType.FIXED;
```

Comment:
Do we need randomization for the pathType value?

Reply:
I have added this.